### DISCUSSION TITLE
Avoid video autoplay

### DISCUSSION THREAD

**Discussion by Nenuial** [quarto-dev/quarto-cli#7889] [Q&A]
### Description

I'm including videos in my revealjs presentation like so : 

```
![](path/to/video)
```

And I have the problem that the video autoplay in the presentation despite having set the YAML option `auto-play-media: false` under the revealjs key.

When I use the speaker notes I have the additional problem that the video plays in both speaker window and main window. 

Any hints as to what I could try to avoid the videos starting automatically?

**Comment by mcanouil** [comment]
I opened an issue to track this at least for the video shortcode:
- #7905 

Now the issue is that you did not provide a reproducible example we can use to reproduce the possible issue with the Reveal.js option `auto-play-media`.
Note that I cannot reproduce your issue with `auto-play-media`.

https://github.com/quarto-dev/quarto-cli/assets/8896044/23a2164c-7109-4390-9da9-67888c1b94d8

**Comment by Nenuial** [comment]
Thank you for your answer. I was so sure that it was just ignorance on my part regarding some setting for autoplay, that I never stop to consider it might be an issue somehow. Sorry about that, I would have made a reproducible example from the start otherwise: <https://github.com/Nenuial/Video>

On further analysis, it apparently has something to do with the template I use ([quarto-revealjs-letterbox](https://github.com/EmilHvitfeldt/quarto-revealjs-letterbox)). Using that template, the videos always autostart (with shortcode or markdown syntax).

I'll investigate a bit more and might open an issue on the template repository.

**Comment by Nenuial** [comment]
There's some JavaScript that autoplays the video in the _quarto-revealjs-letterbox_. I'm sorry I haven't discovered that before.

**Reply by mcanouil** [reply]
The template makes this on purpose:
- https://github.com/EmilHvitfeldt/quarto-revealjs-letterbox/blob/bd776bda50bed09c67e9b33bb27d70739a04b62d/_extensions/letterbox/theme.html#L86


Thus the importance to always provide a reproducible example when asking a question.
Note that if no reproducible example is needed, it's likely when the search on the documentation website can show the answer.

**Reply by Nenuial** [reply]
You're right of course.
